src/Cake.Core.Tests/Unit/IO/NuGet/PackageReferenceBundlerTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Versioning;
+using Cake.Core.IO;
+using Cake.Core.IO.NuGet;
+using NSubstitute;
+using Xunit;
+
+namespace Cake.Core.Tests.Unit.IO.NuGet
+{
+    public sealed class PackageReferenceBundlerTests
+    {
+        public sealed class TheParseMethod
```

Comment:
We can wait with this until later. As long as the tests are there we can do
refactoring of them later.
On Nov 9, 2015 17:53, "James Nail" <notifications@github.com> wrote:

> In src/Cake.Core.Tests/Unit/IO/NuGet/PackageReferenceBundlerTests.cs
> <https://github.com/cake-build/cake/pull/435#discussion_r44298753>:
>
> > @@ -0,0 +1,96 @@
> > +﻿using System;
> > +using System.Collections.Generic;
> > +using System.Linq;
> > +using System.Runtime.Versioning;
> > +using Cake.Core.IO;
> > +using Cake.Core.IO.NuGet;
> > +using NSubstitute;
> > +using Xunit;
> > +
> > +namespace Cake.Core.Tests.Unit.IO.NuGet
> > +{
> > +    public sealed class PackageReferenceBundlerTests
> > +    {
> > +        public sealed class TheParseMethod
>
> I'll give this a try -- some of the fixtures tend to have the opposite
> effect though (sometimes hiding meaningful details in another file).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cake-build/cake/pull/435/files#r44298753>.
>